config/locales/en_radio_elements.yml

```diff
@@ -5,11 +5,9 @@ en:
       default:
         title: Default
         description: |
-          The wind and solar curves for the start year
-          are based on actual production curves of ENTSO-e. The temperature
-          curve is based on KNMI-data.
-          (Full load hours: wind onshore = 1920, wind coastal = 2550, wind offshore = 3500,
-          solar = 867)
+          By default the ETM uses production curves from the start year of your scenario.
+          The corresponding full load hours are shown on this page. You can have a look at the
+          production curves in the <a href='https://pro.energytransitionmodel.com/scenario/flexibility/curve_upload/upload-curves'>Modify profiles</a> section.
```

Comment:
It might be useful to use a relative hyperlink here--i.e., a hyperlink in which the hard-coded "pro.energytransitionmodel.com/..." isn't included. Now, when testing this PR locally, the hyperlink directs me to a different scenario because it points at pro instead of my localhost.

Reply:
Good point, I'll adjust it